modules/mobile/app/controllers/mobile/v0/push_notifications_controller.rb

```diff
@@ -15,22 +14,30 @@ def register
           params[:device_name] || params[:osName]
         )
 
+        Rails.logger.info('VEText Push service register success', app_name: params[:app_name])
```

Comment:
Since these log lines in the controller are logging on the service success could they be in the service itself? (for `get_prefs`, `set_pref`, and `send_notification` too) 

The controller has an automated log line for each action so we have insight on its status.